src/controllers/index.js

```diff
@@ -4,7 +4,10 @@ const router = express.Router();
 
 const homeManager = require('./homeManager');
 
+const supports = require('./support');
+
 router.get('/', homeManager.get);
 
+router.get('/support', supports.get);
```

Comment:
You told me that I should use only what I need and the( get method) is just what was I need in this rout

Reply:
Yeah. This is just an example of how it would look like when things get big.
So, here's how this one should look like:
```javascript
router.route('/support')
  .get(supports.get)